### ISSUE TITLE
Allow overriding help message for custom commands

### ISSUE THREAD

**Issue by mmafic** [nushell/nushell#11149]
### Related problem

I am trying to wrap a Python script that has a complex help output because it uses arguments that work differently from conventional flags and parameters. For example, you can pipe json into it and use arguments like key:=otherkey to map a key in the input to a different key. I don't even know if it's possible to generate help for it using nushell, but even if it is, I would prefer a simple way of passing --help through to my script instead of having to write and maintain many lines of nushell code for something that's already handled inside the script.

### Describe the solution you'd like

Maybe add a `--no-help` flag to `def`, just like there is already `def --wrapped`, and when the `--no-help` flag is present, `--help` is treated like any other flag to your command and can be passed through to the external program.

### Describe alternatives you've considered

Currently I'm just outputting a message to use `--show-help` instead of `--help` as part of the help message auto-generated by NuShell. It's not ideal.

**Comment by kubouch** [comment]
I'm thinking that `def --wrapped` wouldn't define the help flag (same as `extern` currently works).

**Comment by amtoine** [comment]
@mmafic 
is this a duplicate of https://github.com/nushell/nushell/issues/11096, in case you missed it? :innocent:

**Comment by KAAtheWiseGit** [comment]
I think this can be closed after #11235, as a duplicate of #11096.

**Comment by kubouch** [comment]
Yeah, seems like `def --wrapped` should now cover it.
